README.md

```diff
@@ -110,3 +110,22 @@ By default the jobs will be submitted to HTCondor. If you want to use a differen
       myStudy.submit(1, 5) # 1 stands for sixtrack job, 5 is trial number 
       myStudy.collect_result(1, 5) # 1 stands for sixtrack job, 5 is trial number 
       ```
+
+## Description for the database tables
+
+There are ten tables in the database:
+
+| Table Name | Description |
+|:------------:|:-----------|
+|**boinc\_vars**| store the config parameters for boinc jobs |
+|**env** | store some general parameters for the jobs, e.g.: madx\_exe, sixtrack\_exe, study\_path |
+|**templates** | store the template files, e.g. mask file of madx job, fort.3 mother file |
+|**preprocess\_wu**| store the machine parameters for scanning and some general information for preprocess job|
+|**preprocess\_task**| store general information for the preprocess task, every submission will be recorded|
+|**oneturn\_sixtrack\_wu**| store the input parameters of oneturn sixtrack job|
+|**oneturn\_sixtrack\_result**| store the result of oneturn sixtrack job|
+|**sixtrack\_wu**| store the sixtrack parameters for scanning and general information for sixtrack job|
+|**sixtrack\_task**| store general information for the sixtrack task, also record every submission|
+|**six\_results**| store the results of sixtrack jobs, for the moment keep same column name with old sixdb|
+
```

Comment:
I think that it would be great also to have an indication of the length of the tables, e.g.:
- `preprocess\_wu` as many lines as the machine configurations;
- `preprocess\_task` at least as many lines as the `preprocess\_wu`;
- `six\_results` as many lines as the points in all scans.

I think that it would be great to have also e brief general description, like disentangling between pre-processing jobs and actual sixtrack jobs, and the difference between the `_wu` and the `_tasks` tables